src/Parser/DeathKnight/Unholy/CombatLogParser.js

```diff
@@ -41,11 +44,12 @@ class CombatLogParser extends CoreCombatLogParser {
     // items
     coldHeart: ColdHeart,
     tier20_2p: Tier20_2p,
+    coldHeart: ColdHeart,
```

Comment:
It would be a good idea to name this coldHeartEfficiency to match the name of the class it comes from, as well as make it clear that this is different than the regular item statistic.

Reply:
That's entirely my bad. I'm a bit new to this so I didn't know my new commits would end up in the Push Request I added before.